proposed/http-message.md

```diff
@@ -1161,10 +1161,14 @@ interface UriInterface
      * Implementations SHOULD restrict values to "http", "https", or an empty
      * string but MAY accommodate other schemes if required.
      *
-     * If no scheme is present, this method MUST return an empty string.
+     * The string MUST omit the closing scheme delimiter (":"), if present.
```

Comment:
@Tobion I chose to remove the verbiage from each of `withQuery()` and `withFragment()`, as normalization is already properly detailed in the `get*()` methods for each component.